vertx-web-graphql/src/main/java/io/vertx/ext/web/handler/graphql/ApolloWSHandler.java

```diff
@@ -73,4 +65,22 @@ static ApolloWSHandler create(GraphQL graphQL) {
   @GenIgnore(GenIgnore.PERMITTED_TYPE)
   ApolloWSHandler dataLoaderRegistry(Function<RoutingContext, DataLoaderRegistry> factory);
 
+  /**
+   * Set interval seconds in which keepAlive messages are sent. If keepAlive is _null_ or
+   * zero, no keepAlive messages are sent.
+   *
+   * @return a reference to this, so the API can be used fluently
+   */
+  @Fluent
+  @GenIgnore(GenIgnore.PERMITTED_TYPE)
+  ApolloWSHandler keepAlive(Long keepAlive);
```

Comment:
Can you move this setting to an options object? It shouldn't change after initial setup.